backend/src/models/db_lock/index.js

```diff
@@ -10,41 +10,35 @@ const addLockByParams = (requestData) => {
 
 // Remove one.
 const removeOne = (requestData) => {
-  const { locked_row_id, locked_table, locked_by } = requestData;
-  return knex(`${dBLockTable} as dbLock`)
-    .where("dbLock.locked_table", locked_table)
-    .where("dbLock.locked_by", locked_by)
-    .where("dbLock.locked_row_id", locked_row_id)
-    .del();
+  const { locked_row_ids, locked_table, locked_by } = requestData;
+  return knex.raw(`
+  DELETE FROM data.db_lock
+  WHERE locked_row_ids && ARRAY [${locked_row_ids}] AND
+  locked_table = '${locked_table}' AND
+  locked_by = '${locked_by}'
```

Comment:
I don't think we should be inserting user-provided data directly into queries like this. We should use Knex functions since they will properly sanitize/escape the data.

Reply:
I agree with @mhaswell-bcgov on this one, 1. we were only to short circuit the reports to use raw as a temporary measure, 2. i think even if you do raw you should be able to sanitize the inputs, at minimum here, but, also in the controller.
@ASpiteri-BCGov what was your reasoning to 'resolve' the issue. Maybe this is a good point that the developers should discuss.